.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -5,8 +5,10 @@ body:
   - type: markdown
     attributes:
       value: |
+        This issue-tracker is meant for issues related to Dependabot's updating logic, a good rule of thumb is that if you have questions about the _diff_ in a PR, it belongs here.
+
+        Issues related to security alerts or Dependency Graph should instead be filed as a [Code Security discussion](https://github.com/orgs/community/discussions/categories/code-security).
```

Comment:
@courtneycl Is `Code Security` the right category for Dependency Graph? or would you prefer users file these discussions under a different category / issue tracker?

Reply:
yes it is!

Reply:
Confirmed via slack that this is the correct category.